packages/@o3r/test-helpers/src/test-environments/create-test-environment-angular.ts

```diff
@@ -133,8 +133,8 @@ export async function createTestEnvironmentAngular(inputOptions: Partial<CreateT
 
     // Run ng-adds
     const project = options.generateMonorepo ? '--project=test-app' : '';
-    packageManagerExec(`ng add @angular/pwa ${project}`, execAppOptions);
-    packageManagerExec(`ng add @angular/material ${project}`, execAppOptions);
+    packageManagerExec({script: 'ng', args: ['add', '@angular/pwa', project]}, execAppOptions);
+    packageManagerExec({script: 'ng', args: ['add', '@angular/material', project]}, execAppOptions);
 
     if (options.generateMonorepo) {
       // TODO remove this when https://github.com/AmadeusITGroup/otter/issues/603 has been resolved
```

Comment:
isn't it fixed?

Reply:
deprecated environement, no fix will be done on it on this PR